src/cli/test/test_cli.c

```diff
@@ -1453,6 +1665,272 @@ static void test_trn_cli_update_packet_metadata_subcmd(void **state)
+		"update_transit_namespace_label_policy succeed with well formed endpoint json input");
+	update_transit_namespace_label_policy_1_ret_val = 0;
+	expect_function_call(__wrap_update_transit_namespace_label_policy_1);
+	will_return(__wrap_update_transit_namespace_label_policy_1, &update_transit_namespace_label_policy_1_ret_val);
+	expect_check(__wrap_update_transit_namespace_label_policy_1, namespace_label_policy, check_namespace_label_policy_equal, &exp_namespace_label_policy);
+	expect_any(__wrap_update_transit_namespace_label_policy_1, clnt);
+	rc = trn_cli_update_transit_namespace_label_policy_subcmd(NULL, argc, argv1);
+	assert_int_equal(rc, 0);
+
+	/* Test parse packet metadata input error*/
+	TEST_CASE("update_transit_namespace_label_policy is not called with non-string field");
+	rc = trn_cli_update_transit_namespace_label_policy_subcmd(NULL, argc, argv2);
+	assert_int_equal(rc, -EINVAL);
+
+	TEST_CASE("update_transit_namespace_label_policy is not called with missing required field");
+	rc = trn_cli_update_transit_namespace_label_policy_subcmd(NULL, argc, argv3);
+	assert_int_equal(rc, -EINVAL);
+
+	/* Test parse packet metadata input error 2*/
+	TEST_CASE("update_transit_namespace_label_policy is not called malformed json");
+	rc = trn_cli_update_transit_namespace_label_policy_subcmd(NULL, argc, argv4);
+	assert_int_equal(rc, -EINVAL);
+
+	/* Test call update_transit_namespace_label_policy_1 return error*/
+	TEST_CASE(
+		"update_transit_namespace_label_policy subcommand fails if update_transit_namespace_label_policy_1 returns error");
+	update_transit_namespace_label_policy_1_ret_val = -EINVAL;
+	expect_function_call(__wrap_update_transit_namespace_label_policy_1);
+	will_return(__wrap_update_transit_namespace_label_policy_1, &update_transit_namespace_label_policy_1_ret_val);
+	expect_any(__wrap_update_transit_namespace_label_policy_1, namespace_label_policy);
+	expect_any(__wrap_update_transit_namespace_label_policy_1, clnt);
+	rc = trn_cli_update_transit_namespace_label_policy_subcmd(NULL, argc, argv1);
+	assert_int_equal(rc, -EINVAL);
+
+	/* Test call update_transit_namespace_label_policy_1 return NULL*/
+	TEST_CASE(
+		"update_transit_namespace_label_policy subcommand fails if update_transit_namespace_label_policy_1 returns NULl");
+	expect_function_call(__wrap_update_transit_namespace_label_policy_1);
+	will_return(__wrap_update_transit_namespace_label_policy_1, NULL);
+	expect_any(__wrap_update_transit_namespace_label_policy_1, namespace_label_policy);
+	expect_any(__wrap_update_transit_namespace_label_policy_1, clnt);
+	rc = trn_cli_update_transit_namespace_label_policy_subcmd(NULL, argc, argv1);
+	assert_int_equal(rc, -EINVAL);
+}
+
+static void test_trn_cli_update_transit_pod_and_namespace_label_policy_subcmd(void **state)
+{
+	UNUSED(state);
+	int rc;
+	int argc = 5;
+	int update_transit_pod_and_namespace_label_policy_1_ret_val = 0;
+
+	/* Test cases */
+	char *argv1[] = { "update-pod-label-policy", "-i", "eth0", "-j", QUOTE([{
```

Comment:
This test can be expanded to pass empty pod / ns label values to the pod_and_ns CLI APIs. It would be much simpler that way.